packages/edit-site/src/components/list/actions/index.js

```diff
@@ -30,16 +29,23 @@ export default function Actions( { template } ) {
 	}
 
 	async function revertAndSaveTemplate() {
+		const noticeId = 'edit-site-template-reverted';
+		removeNotice( noticeId );
 		try {
 			await revertTemplate( template, { allowUndo: false } );
 			await saveEditedEntityRecord(
 				'postType',
 				template.type,
 				template.id
 			);
+			const notice =
+				template.type === 'wp_template'
+					? __( 'Template reverted.' )
+					: __( 'Template part reverted.' );
 
-			createSuccessNotice( __( 'Entity reverted.' ), {
+			createSuccessNotice( notice, {
 				type: 'snackbar',
+				id: noticeId,
```

Comment:
Yeh, my thinking in this case was that because the unique id prevents the stacking of snackbars, if you revert multiple templates one after the other the user doesn't get a clear indication of the success of each revert, ie. they revert one and the snackbar appears and stays, they revert another and the same snackbar is still just sitting there. But with the remove added a new snackbar appears for each revert to give confirmation, but still only one at a time.

Do you think this makes sense?